src/Microsoft.ML.ImageAnalytics/Properties/AssemblyInfo.cs

```diff
@@ -6,5 +6,6 @@
 using Microsoft.ML;
 
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.StaticPipe" + PublicKey.Value)]
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Tests" + PublicKey.TestValue)]
```

Comment:
[https://github.com/dotnet/machinelearning/issues/2306](https://github.com/dotnet/machinelearning/issues/2306)
Maybe we should convert all direct calls to estimators/transformers to calls to mlContext extensions?
feel free to ignore.

Reply:
are we doing that now, or post V1? i agree, but might make more sense to spend the time in other things and do this later. 

---
In reply to: [253244221](https://github.com/dotnet/machinelearning/pull/2372#discussion_r253244221) [](ancestors = 253244221)